estate/models/estate_property.py

```diff
@@ -0,0 +1,153 @@
+from markupsafe import Markup
+from dateutil.relativedelta import relativedelta
+
+from odoo import _, api, fields, models
+
+
+from odoo.exceptions import UserError, ValidationError
+from odoo.tools import float_compare, float_is_zero
+
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "Estate Property"
+    _inherit = ["mail.thread", "mail.activity.mixin"]
+    _order= "id desc"
+    
+    name = fields.Char(string="Title", required=True)
+    description = fields.Text(string="Description")
+    postcode = fields.Char(string="Postcode")
+    date_availability = fields.Date(
+        string="Available From", 
+        copy=False, 
+        default=lambda self: fields.Date.today() + relativedelta(months=3)
+    )
+    expected_price = fields.Float(string="Expected Price", required=True)
+    selling_price = fields.Float(string="Selling Price", readonly=True, copy=False)
+    bedrooms = fields.Integer(string="Bedrooms", default=2)
+    living_area = fields.Integer(string="Living Area (sqm)")
+    facades = fields.Integer(string="Facades")
+    garage = fields.Boolean(string="Garage")
+    garden = fields.Boolean(string="Garden")
+    garden_area = fields.Integer(string="Garden Area (sqm)")
+    garden_orientation = fields.Selection(
+        string="Garden Orientation",
+        selection=[
+            ('north', 'North'),
+            ('south', 'South'),
+            ('east', 'East'),
+            ('west', 'West')
```

Comment:
```suggestion
            ('north', "North"),
            ('south', "South"),
            ('east', "East"),
            ('west', "West")
```
strings visible to user always write it in `""`